class XeroAPIException(ValueError):
    """
    An error occurred while interacting with the Xero API.
    """
